servant-mock/test/Servant/MockSpec.hs

```diff
@@ -40,8 +38,11 @@ data TestHeader
   | ArbitraryHeader
   deriving (Show)
 
-instance ToByteString TestHeader where
-  builder = fromString . show
+instance ToHttpApiData TestHeader where
+  toHeader = toHeader . show
+  toUrlPiece _ = error "ToHttpApiData.toUrlPiece not implemented for TestHeader"
```

Comment:
I assumed the intent of TestHeader was only to test headers - happy to add the trivial instances if you'd prefer, but it didn't make much sense to me.